src/main/java/org/pih/petl/api/JobExecution.java

```diff
@@ -2,51 +2,59 @@
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
 import javax.persistence.Id;
 import java.util.Date;
-
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
+import java.util.UUID;
 
 /**
  * Represents an ETL job execution and the status of this
  */
 @Entity(name = "petl_job_execution")
 public class JobExecution {
 
-    private static Log log = LogFactory.getLog(JobExecution.class);
-
     @Id
     private String uuid;
 
-    @Column(name = "job_path", nullable = false, length = 100)
+    @Column(name = "job_path", length = 1000)
     private String jobPath;
 
-    @Column(name = "total_expected")
-    private Integer totalExpected;
+    @Column(name ="parent_execution_uuid", length = 36)
+    private String parentExecutionUuid;
+
+    @Column(name = "description", length = 1000)
+    private String description;
```

Comment:
This adds a description for the job, which is configured in the job configuration as a top-level attribute.  It is optional.